modules/mobile/app/models/mobile/v0/adapters/claims_overview.rb

```diff
@@ -10,7 +10,7 @@ def parse(entry)
                              id: entry['evss_id'].to_s,
                              type: 'claim',
                              subtype: entry['list_data']['status_type'],
-                             completed: entry['list_data']['claim_status'] != 'PEND',
+                             completed: entry['list_data']['status'] != 'COMPLETE',
```

Comment:
```suggestion
                             completed: entry['list_data']['status'] == 'COMPLETE',
```

Should the completed key here but based on if the status is NOT complete?

Reply:
Whoops, thanks. Also I'll add a few more checks to the test cases so something like that can't slip though.